src/overmap.cpp

```diff
@@ -2157,7 +2157,11 @@ tripoint overmap::draw_overmap(const tripoint &orig, const draw_data_t &data)
             const std::string new_note = string_input_popup(
                 _("Note (X:TEXT for custom symbol, G; for color):"),
                 45, old_note, color_notes); // 45 char max
-            if(old_note != new_note) {
+            if(new_note == "" && old_note != "") {
```

Comment:
that magic string constant is what string_input_popup returns when you hit escape